### PR TITLE
Update licensing to AGPL

### PR DESCRIPTION
**Pull request by tomberek**
This is meant to capture the discussion for a move to GPLv3 for this repo.

Signed-off-by: Tom Bereknyei <tom@dds.mil>

### CODE REVIEW

**Comment by nacin** [comment]
@tomberek @nickjs @BrandonBouier Was it intentional to go with AGPL over GPL? There seems to be no context/discussion on this PR about that, so I'm curious if that discussion happened elsewhere. GPL would also not allow someone to fork, improve, and **release** (key word) as closed-source, but doesn't come with the extra baggage of the Affero.

For what it's worth, jQuery was dual MIT/GPL for one reason only: Drupal didn't want to include non-GPL code in its distribution (even though you can include MIT libraries in a GPL distribution) and so they asked for the dual license. GPL [was dropped](https://blog.jquery.com/2012/09/10/jquery-licensing-changes/) back in 2012.

**Comment by nacingov** [comment]
@tomberek @nickjs @BrandonBouier Was it intentional to go with AGPL over GPL? There seems to be no context/discussion on this PR about that, so I'm curious if that discussion happened elsewhere. GPL would also not allow someone to fork, improve, and release (key word) as closed-source, but doesn't come with the extra baggage of the Affero.

For what it's worth, jQuery was dual MIT/GPL for one reason only: Drupal didn't want to include non-GPL code in its distribution (even though you can include MIT libraries in a GPL distribution) and so they asked for the dual license. GPL was dropped back in 2012.

**Comment by BrandonBouier** [comment]
@nacingov yes it was intentional.

In fact, it was the FSF who recommended we use the AGPL over the GPL for this project (see http://www.fsf.org/blogs/licensing/the-licensing-and-compliance-lab-interviews-defense-digital-service-about-emcm). At the end of the day though, the license decision was left up to the project developers (@tomberek and @nickjs).

The only "baggage" I'm personally aware of with the AGPL tends to affect private businesses who do not wish to share the source behind their SaaS systems. If there are other issues that you know of, please let us know.

**Comment by tomberek** [comment]
tl;dr We are experimenting

@nacingov To add to Brandon's response, this is also a small enough project that it would be relatively easy to recreate it, and improve upon it as well. Whatever extra baggage seems to be fine compared to the proof-of-concept that the projects can join copyleft communities. The specific point of AGPL ensure a hosted copy served to the public also releases its source. This is not true for GPL and is the whole reason AGPL exists. 

Do you consider the AGPL requirements onerous in this case?
